internal/controller/platform/services/services.go

```diff
@@ -89,6 +97,8 @@ type PlatformServiceHandler interface {
 	IsServiceSetInSpec() bool
 	// IsServiceEnabledInSpec returns true if the service is enabled in the spec.
 	IsServiceEnabledInSpec() bool
+	// IsPersistenceSetInSpec returns true if the service has persistence set in the spec.
+	IsPersistenceSetInSpec() bool
```

Comment:
```suggestion
	IsPersistenceEnabledtInSpec() bool
```
For ubiquity within the interface.